Eat_and_Greet/src/comps/ChatCard/ChatCard.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { View, Text, TouchableOpacity } from 'react-native';
+import Icon from 'react-native-vector-icons/MaterialCommunityIcons';
+
+import localstyle from './styles';
+
+
+interface ChatCardProps {
+  senderName: string;
+  message: string;
+  timestamp: string;
+  onPress?: () => void;
+  isRead?: boolean;
+
+}
+
+const ChatCard = ({ senderName, message, timestamp, onPress, isRead} : ChatCardProps) => {
```

Comment:
Still think we should change 'senderName' prop name. Nitpick tho and not necessary now.